packages/core/src/editor/EditorPopupMenu.ts

```
@@ -190,22 +191,22 @@ export class EditorPopupMenu {
       if (item.nodeName === 'add') {
         const condition = item.getAttribute('if');
 
-        if (condition == null || conditions[condition]) {
+        if (isNullish(condition) || conditions[condition]) {
           let as = item.getAttribute('as')!;
           as = Translations.get(as) || as;
           const funct = eval(getTextContent(<Text>(<unknown>item)));
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)